### ISSUE TITLE
Using webhook to send alert to Rocket chat leads to empty message in Rocket

### ISSUE THREAD

**Issue by patrijua** [SigNoz/signoz#2224]
## Bug description

New alert channel is created to use webhook to send message to Rocket chat. When alert is firing in SigNoz, there comes only empty message to Rocket channel with no content. There are no errors or warnings on SigNoz alertmanager logs.

<img width="619" alt="image" src="https://user-images.githubusercontent.com/57624530/218112366-7b4dfba1-c78b-4ded-80a1-91ae7eab4ea4.png">

## Expected behavior

I would expect the message to contain payload described in https://signoz.io/docs/userguide/alerts-management/#d-receive-alert-through-webhook

## How to reproduce

1. Configure webhook in Rocket chat
2. Create a new alarm channel using the created webhook
3. Create an new alarm using the created channel
4. Trigger the alarm -> empty message in Rocket

## Version information 
* **Signoz version**: v0.15.0

**Comment by adolfombarros** [comment]
**Solution: Using RocketChat Webhook Script with SigNoz**

The SigNoz Slack channel type was causing errors because it expected Slack API responses. Switching to SigNoz's **Webhook channel type** with a custom RocketChat incoming webhook script resolves this issue.

**Implementation:**
1. Create an incoming webhook integration in RocketChat with script enabled
2. Add the JavaScript processing script to transform SigNoz's JSON payload into RocketChat's message format
3. Configure SigNoz webhook channel to use the RocketChat webhook URL

The script processes SigNoz alert data and formats it with proper attachments, colors, and fields for RocketChat display. Notifications now deliver successfully without API compatibility errors.

```
class Script {
  process_incoming_request({ request }) {
    // Dados vindos do SigNoz
    const data = request.content;
    
    // Formata a mensagem para o RocketChat
    let text = `[${data.status ? data.status.toUpperCase() : 'ALERT'}] ${data.commonLabels?.alertname || 'Alert'}`;
    
    let attachments = [];
    
    if (data.alerts && Array.isArray(data.alerts)) {
      data.alerts.forEach(alert => {
        attachments.push({
          color: alert.status === 'firing' ? '#FF0000' : '#00FF00',
          title: alert.labels?.alertname || 'Alert',
          fields: [
            {
              title: 'Severity',
              value: alert.labels?.severity || 'N/A',
              short: true
            },
            {
              title: 'Summary',
              value: alert.annotations?.summary || 'N/A',
              short: false
            },
            {
              title: 'Description',
              value: alert.annotations?.description || 'N/A',
              short: false
            }
          ]
        });
      });
    }
    
    return {
      content: {
        text: text,
        attachments: attachments
      }
    };
  }
}
```

**Comment by lustsazeus-lab** [comment]
Good catch — empty Rocket.Chat messages here often come from payload shape mismatches (text/attachments/card fields) even when webhook delivery itself succeeds.\n\nCould you share one sanitized outbound payload + Rocket.Chat incoming webhook settings? With that I can propose a minimal normalization patch and test case.\n\nHappy to help push a focused 48h fix PR if you want this unblocked quickly.

**Comment by Samyra312007** [comment]
Respected Maintainers, If this issue is still open then i would like to work on it.
